runtime/binding-mqtt-kafka/src/main/java/io/aklivity/zilla/runtime/binding/mqtt/kafka/internal/config/MqttKafkaConditionMatcher.java

```diff
@@ -14,42 +14,79 @@
  */
 package io.aklivity.zilla.runtime.binding.mqtt.kafka.internal.config;
 
+import java.util.ArrayList;
+import java.util.List;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
+import java.util.stream.Collectors;
 
 import io.aklivity.zilla.runtime.binding.mqtt.kafka.config.MqttKafkaConditionConfig;
 
 public class MqttKafkaConditionMatcher
 {
-    private final Matcher topicMatcher;
-    public final String topic;
+    public final List<Matcher> subscribeMatchers;
```

Comment:
We're checking if null in matchesClient which is in the route config, so that we only check try to bootstrap in routes where there is at least one subscribe "when".
Should I replace it with a getter though?